packages/webapp/src/components/MultiSelect.tsx

```diff
@@ -12,9 +12,10 @@ export interface MultiSelectArgs<T> {
     defaultSelect: T[];
     all?: boolean;
     onChange: (selected: T[]) => void;
+    emptyLabel?: string;
 }
 
-export const MultiSelect: React.FC<MultiSelectArgs<any>> = ({ label, options, selected, defaultSelect, all, onChange }) => {
+export const MultiSelect: React.FC<MultiSelectArgs<any>> = ({ label, options, selected, defaultSelect, all, onChange, emptyLabel = 'No framework found.' }) => {
```

Comment:
thanks 👌🏻 I would go even further and remove the default (I made the component and it was later moved to be reusable but this details slipped)